.gitignore

```diff
@@ -3,4 +3,5 @@
 .vscode
 *.test
 *.ign
+go.work
```

Comment:
Why was removed?

Reply:
It seems if not necessary for the repo to have a `go.work`, so it was renamed to `go.work.example` to make the workspace optional to users